otcextensions/sdk/modelartsv1/v1/service_log.py

```diff
@@ -0,0 +1,64 @@
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+from openstack import resource
+from otcextensions.sdk.modelartsv1.v1.service import ConfigSpec
+
+
+class ResultSpec(resource.Resource):
+    #: Node ID.
+    node_id = resource.Body("node_id")
+    #: Node name.
+    node_name = resource.Body("node_name")
+    #: Operation type.
+    operation = resource.Body("operation")
+    #: Operation result.
+    result = resource.Body("result", type=bool)
+
+
+class ServiceLog(resource.Resource):
+    base_path = "/services/%(serviceId)s/logs"
+
+    resources_key = "logs"
+
+    allow_list = True
+
+    _query_mapping = resource.QueryParameters(
+        "update_time",
+    )
+
+    #: Service ID.
+    serviceId = resource.URI("serviceId")
```

Comment:
service_id

Reply:
serviceId - resource.URI # base_path

Reply:
you can use query parameters aliases, dont you?

Reply:
updated